projects/core/src/cms/cms.module.ts

```diff
@@ -18,7 +40,10 @@ export class CmsModule {
   static forRoot(): ModuleWithProviders<CmsModule> {
     return {
       ngModule: CmsModule,
-      providers: [CmsService, provideDefaultConfig(defaultCmsModuleConfig)],
+      providers: [
+        CmsService,
+        provideDefaultConfigFactory(getDefaultCmsConfig, [ConfigChunk]),
```

Comment:
I think we can skip couple of if's and move a logic directly here. What do you think?

```
@NgModule({
  imports: [CmsStoreModule, PageMetaModule.forRoot()],
})
export class CmsModule {
  static forRoot(): ModuleWithProviders<CmsModule> {
    return {
      ngModule: CmsModule,
      providers: [
        CmsService,
        provideDefaultConfigFactory(
          (configChunks) => {
            configChunks.find((config: Config) => {
              if (Boolean(config.features?.[USER_CMS_ENDPOINTS])) {
                return defaultUserCmsModuleConfig;
              }
            });

            return defaultCmsModuleConfig;
          },
          [ConfigChunk]
        ),
      ],
    };
  }
}
```